### PR TITLE
Update uncle detail page 

### PR DESCRIPTION
**Pull request by gesseekur**
https://app.asana.com/0/1126491781150662/1178123223053659

### CODE REVIEW

File: newclient/src/modules/uncles/pages/PageDetailsUncle.vue [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+@Component({
+    components: {
+        AppBreadCrumbs,
+        AppDetailsList
+    },
+    apollo: {
+        uncleDetail: {
+            query() {
+                const self = this as any
```
Comment (olgakup) [inline newclient/src/modules/uncles/pages/PageDetailsUncle.vue:29]: Is this from  the old files?
can you look at BlockDetailsPage and Block Details Handler? and follow similar structure.

File: newclient/src/modules/uncles/handlers/UncleDetails.vue [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        } else {
+            // not too sure if we still need this check
+            if (!this.uncle) {
+                this.error = this.$i18n.t('message.invalid.uncle').toString()
+                return []
+            }
+            details = [
+                {
+                    title: this.$i18n.t('uncle.height'),
+                    detail: this.uncle.block.summary.number
```
Comment (olgakup) [inline newclient/src/modules/uncles/handlers/UncleDetails.vue:118]: This numbers should be formatted  with the Number Formatter Mixin

Comment (olgakup) [inline newclient/src/modules/uncles/handlers/UncleDetails.vue:118]: formatNumber()

File: newclient/src/modules/uncles/handlers/UncleDetails.vue [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                    detail: this.$i18n.d(this.uncle.block.summary.timestamp, 'long', this.$i18n.locale.replace('_', '-'))
+                },
+                {
+                    title: this.$i18n.t('common.sha'),
+                    detail: this.uncle.block.sha3Uncles,
+                    mono: true
+                },
+                {
+                    title: this.$i18n.t('gas.limit'),
+                    detail: this.uncle.block.gasLimit
```
Comment (olgakup) [inline newclient/src/modules/uncles/handlers/UncleDetails.vue:158]: formatNumber()

File: newclient/src/modules/uncles/handlers/UncleDetails.vue [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                    // tooltip: this.uncle.gasLimit.tooltipText ? `${this.uncle.gasLimitFormatted.tooltipText}` : undefined
+                },
+                {
+                    title: this.$i18n.t('gas.used'),
+                    detail: this.uncle.block.gasUsed
```
Comment (olgakup) [inline newclient/src/modules/uncles/handlers/UncleDetails.vue:163]: formatNumber()

File: newclient/src/modules/uncles/handlers/UncleDetails.vue [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+                // },
+                {
+                    title: this.$i18n.t('common.hash'),
+                    detail: this.uncle.block.hash,
+                    copy: true,
+                    mono: true
+                },
+                {
+                    title: this.$i18n.t('block.p-hash'),
+                    detail: this.uncle.parentHash,
```
Comment (olgakup) [inline newclient/src/modules/uncles/handlers/UncleDetails.vue:137]: add link

Comment (gesseekur) [reply newclient/src/modules/uncles/handlers/UncleDetails.vue:138]: added link!

File: newclient/src/modules/uncles/handlers/UncleDetails.vue (hunk 4 above)
Comment (gesseekur) [reply newclient/src/modules/uncles/handlers/UncleDetails.vue]: done

File: newclient/src/modules/uncles/handlers/UncleDetails.vue (hunk 3 above)
Comment (gesseekur) [reply newclient/src/modules/uncles/handlers/UncleDetails.vue]: done

File: newclient/src/modules/uncles/handlers/UncleDetails.vue (hunk 2 above)
Comment (gesseekur) [reply newclient/src/modules/uncles/handlers/UncleDetails.vue]: done

**Pull request merged**